flow/templates/bridges.sh

```diff
@@ -60,7 +60,4 @@
 {% if account %}
 #SBATCH -A {{ account }}
 {% endif %}
-{% if memory %}
```

Comment:
Why have we removed this part of the jobscript without replacing it with something else? We still need to be able to request certain amounts of memory right?

Reply:
It's handled by the memory snippet in the parent template instead of by each cluster: https://github.com/glotzerlab/signac-flow/blob/1b2d44ea088e812f96da6ccb3521b70a04ecc009/flow/templates/slurm.sh#L8-L10

Reply:
@tommy-waltmann just in case you were curious, that line was added to slurm.sh by #256, which was a recent PR that this one depends on. Since the changes got split into two PRs, it's not obvious from this PR that we're not just removing the feature.